### ISSUE TITLE
Significant Sass build time difference between Nuxt and Vite

### ISSUE THREAD

**Issue by miosenpai** [nuxt/nuxt#20533]
### Environment

```
------------------------------
- Operating System: Linux
- Node Version:     v16.14.2
- Nuxt Version:     3.4.2
- Nitro Version:    2.3.3
- Package Manager:  npm@7.17.0
- Builder:          vite
- User Config:      -
- Runtime Modules:  -
- Build Modules:    -
------------------------------
```

### Reproduction

Nuxt Project: https://stackblitz.com/edit/github-4zwpte
Vite Project: https://stackblitz.com/edit/github-soapej

### Describe the bug

While working on a Nuxt project, I noticed the project's start up time increased significantly (from a few hundred ms to 9000ms+) after I imported a Sass library. I originally thought this was from the Sass compiler being slow in general but after replicating the same setup in a standalone Vite project I saw very little increase to its start up time, leading me to consider if the performance hit could be Nuxt specific.

Something else I noticed is that with `DEBUG=*`, the Nuxt project logged additional `.scss` entries and the entries also took more time compared to the Vite project (although this could be intended behavior?).

### Additional context

I tried disabling CSS sourcemaps in the Nuxt project after finding https://github.com/vitejs/vite/issues/8656#issuecomment-1160683426 using `sourcemap: false` in `nuxt.config.ts` (https://github.com/nuxt/nuxt/pull/18446), but this didn't seem to help.

I was also able to reproduce the slowdown with other Sass libraries like `bootstrap/scss/bootstrap.scss`.

**Comment by PixsaOJ** [comment]
So is this issue fixed? Or do we know the problem?

**Comment by manniL** [comment]
Should be fixed since 3.8.1, yes.

**Comment by PixsaOJ** [comment]
Well, I am on  "3.8.2" soo...
